### PR TITLE
Update components in settings doc

### PR DESCRIPTION
**Pull request by fedeericodl**
Added `CheckboxItem`, `SliderItem` and `TextArea`. All other component props have also been updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,18 +22,18 @@
...
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.3.0",
-    "@docusaurus/theme-classic": "^2.3.0",
-    "@docusaurus/types": "^2.3.0",
+    "@docusaurus/module-type-aliases": "2.3.1",
+    "@docusaurus/theme-classic": "^2.3.1",
+    "@docusaurus/types": "^2.3.1",
```
Comment (asportnoy) [inline package.json]: Can you makeall of the docusaurus stuff allow compatible versions? (`^` prefix)

Comment (fedeericodl) [reply package.json]: Done

File: .vscode/settings.json [hunk 2]
```diff
@@ -2,6 +2,6 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "yaml.schemas": {
-    "https://json.schemastore.org/github-workflow.json": "file:///Users/albert/Programming/Discord/replugged-guide/.github/workflows/cspell.yml"
```
Comment (asportnoy) [inline .vscode/settings.json:5]: oops lol

**Review by asportnoy — APPROVED** [review]

**Comment by asportnoy** [comment]
@allcontributors please add @FedeIlLeone for docs

**Pull request merged**